install.md

```diff
@@ -195,6 +195,96 @@ echo 'deb [signed-by=/usr/share/keyrings/projectatomic-ppa.gpg] http://ppa.launc
 apt update
 apt -y install -t stretch-backports libostree-dev golang
 apt -y install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
-``` 
+```
 
 The build steps on Debian are otherwise the same as Ubuntu, above.
+
+## Configuration files
+
+### [registries.conf](https://github.com/containers/image/blob/master/docs/registries.conf.5.md)
+
+`/etc/containers/registries.conf`
+
+registries.conf is the configuration file which specifies which container registries should be consulted when completing image names which do not include a registry or domain portion.
+
+Example from Fedora containers-common package
+
+```
+cat /etc/containers/registries.conf
```

Comment:
I'm concerned about having the contents of the files in here and the contents becoming out of date and not changed on this page.  Could we instead add a link to the file in their GitHub project?  If you're really married to keeping the contents here, could you add a link to the file anyway and perhaps a note that file location is the most current?

Reply:
These files are in the lookaside cache, Not currently stored in an upstream repo at this time.  If we created a containers-common package and put these contents in there, I would be fine, with it.  But for now this is the best we got.